kotest-assertions/kotest-assertions-core/src/jvmTest/kotlin/com/sksamuel/kotest/assertions/timing/EventuallyTest.kt

```diff
@@ -46,14 +46,14 @@ class EventuallyTest : WordSpec() {
          }
          "pass tests that completed within the time allowed" {
             val end = System.currentTimeMillis() + 150
-            eventually(1.seconds) {
+            eventually(1.seconds, RuntimeException::class) {
```

Comment:
Would this test fail now if we didn't add the exception ?
Will it break existing users code in other words ?

Reply:
Yes, it will. Other proposed solution might be to add `Throwable` to configuration's default constructor. Then it shouldn't break existing code, so changes to tests might be reverted. Should I do it?